OWASP dependency-check throwing error when downloading data from NVD

### Checklist

- [x] I confirm there are no unresolved issues reported on the [Chocolatey Status page](https://status.chocolatey.org).
- [x] I have verified this is the correct repository for opening this issue.
- [x] I have verified no other issues exist related to my problem.
- [x] I have verified this is not an issue for a specific package.
- [x] I have verified this issue is not security related.
- [x] I confirm I am using **official**, and **not** unofficial, or modified, Chocolatey products.

### What You Are Seeing?

When executing the SonarQube and Dependency-Check tasks, the following error is being output when vulnerability data from NVD and the build fails.

```
22:08:46       [INFO] Checking for updates
22:08:49       [ERROR] Error updating the NVD Data
22:08:49       org.owasp.dependencycheck.data.update.exception.UpdateException: Error updating the NVD Data
22:08:49         at org.owasp.dependencycheck.data.update.NvdApiDataSource.processApi(NvdApiDataSource.java:399)
22:08:49         at org.owasp.dependencycheck.data.update.NvdApiDataSource.update(NvdApiDataSource.java:117)
22:08:49         at org.owasp.dependencycheck.Engine.doUpdates(Engine.java:903)
22:08:49         at org.owasp.dependencycheck.Engine.initializeAndUpdateDatabase(Engine.java:708)
22:08:49         at org.owasp.dependencycheck.Engine.analyzeDependencies(Engine.java:634)
22:08:49         at org.owasp.dependencycheck.App.runScan(App.java:269)
22:08:49         at org.owasp.dependencycheck.App.run(App.java:201)
22:08:49         at org.owasp.dependencycheck.App.main(App.java:93)
22:08:49       Caused by: io.github.jeremylong.openvulnerability.client.nvd.NvdApiException: Failed to parse NVD data
22:08:49         at io.github.jeremylong.openvulnerability.client.nvd.NvdCveClient._next(NvdCveClient.java:389)
22:08:49         at io.github.jeremylong.openvulnerability.client.nvd.NvdCveClient.next(NvdCveClient.java:357)
22:08:49         at org.owasp.dependencycheck.data.update.NvdApiDataSource.processApi(NvdApiDataSource.java:355)
22:08:49         ... 7 common frames omitted
22:08:49       Caused by: com.fasterxml.jackson.databind.exc.ValueInstantiationException: Cannot construct instance of `io.github.jeremylong.openvulnerability.client.nvd.CvssV4Data$ModifiedCiaType`, problem: SAFETY
```

### What is Expected?

NVD data is downloaded and parsed successfully.

### How Did You Get This To Happen?

Execute a build that is configured to execute SonarQube and Dependency-Check tasks. The error is generated at the start of the Dependency-Check task.

### System Details

N/A

### Installed Packages

```bash
N/A
```

### Output Log

```bash
N/A
```

### Additional Context

https://github.com/dependency-check/DependencyCheck/issues/7406

--- a/Chocolatey.Cake.Recipe/Content/dependencyCheck.cake
+++ b/Chocolatey.Cake.Recipe/Content/dependencyCheck.cake
@@ -22,7 +22,7 @@ BuildParameters.Tasks.DependencyCheckTask = Task("Dependency-Check")
     .Does(() => RequireTool(ToolSettings.DependencyCheckTool, () =>
 {
     DownloadFile(
-        "https://github.com/jeremylong/DependencyCheck/releases/download/v12.0.1/dependency-check-12.0.1-release.zip",
+        "https://github.com/dependency-check/DependencyCheck/releases/download/v12.1.0/dependency-check-12.1.0-release.zip",
         BuildParameters.RootDirectoryPath.CombineWithFilePath("dependency-check.zip")
     );
 

